packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/identity-client.test.ts

```diff
@@ -82,13 +113,46 @@ test("identity-clients", async (t: Test) => {
         logLevel: logLevel,
       }),
     );
+
+    console.log(vaultIpAddr);
```

Comment:
@brioux No console logging please, use the logger from the common package

Reply:
@petermetz replaced all console logs